main.go

```diff
@@ -37,10 +37,12 @@ var (
 		"the gcp project number. If unknown, can be found via 'gcloud projects list'")
 	vpcNetworkName     = flag.String("vpc-network-name", "default", "VPC network name")
 	localityZone       = flag.String("locality-zone", "", "the locality zone to use, instead of retrieving it from the metadata server. Useful when not running on GCP and/or for testing")
-	includeV3Features  = flag.Bool("include-v3-features-experimental", false, "whether or not to generate configs which works with the xDS v3 implementation in TD. This flag is EXPERIMENTAL and may be changed or removed in a later release.")
```

Comment:
Leave the old flag and just change the default value. That prevents needing to do an atomic upgrade (e.g., to upgrade the current v3 tests), and is easier on us poor humans.

Reply:
Done. Thanks for the suggestion.